README.md

```diff
@@ -13,6 +13,25 @@ Note: There is a tag mdspan-0.4.0 which reflects the status of P0009 before
 * renaming `pointer`, `data`, `is_contiguous` and `is_always_contiguous`; and before
 * renaming `size_type` to `index_type` and introducing a new `size_type = make_unsigned_t<index_type>` alias.
 
+Building code with `mdspan`
+---------------------------
+
+mdspan is discoverable by CMake:
+
+```
+find_package( mdspan REQUIRED )
+target_link_libraries( myproject PUBLIC std::mdspan )
+```
+
+Your code can then have
+```
+#include "mdspan/mdspan.hpp"
+namespace md = Kokkos;
+...
+md::mdspan
```

Comment:
I think I would just have the include here (I personally prefer with angle brackets, but I suppose it's a matter of taste)

```suggestion
```cpp
#include <mdspan/mdspan.hpp>
```

I think we could probably leave out the namespace alias here